ewitem.py

```diff
@@ -1628,3 +1629,31 @@ def remove_from_trades(id_item):
 
 	await ewutils.send_message(cmd.client, cmd.message.channel, ewutils.formatMessage(cmd.message.author, response))
 
+def surrendersoul(giver = None, receiver = None, id_server=None):
+	print("DEAD\n\n\n\n\n\n\n\n\n")
+	if giver != None and receiver != None:
+		receivermodel = EwUser(id_server=id_server, id_user=receiver)
+		givermodel = EwUser(id_server=id_server, id_user=giver)
+		giverplayer = EwPlayer(id_user=givermodel.id_user)
+
+		givermodel.has_soul = 0
```

Comment:
we should definitely check if the user still has their soul before potentially creating a duplicate